packages/statemanager/src/ethersStateManager.ts

```diff
@@ -34,12 +45,10 @@ export class EthersStateManager implements EVMStateManagerInterface {
       typeof window === 'undefined' ? process?.env?.DEBUG?.includes('ethjs') ?? false : false
 
     this._debug = createDebugLogger('statemanager:ethersStateManager')
-    if (typeof opts.provider === 'string') {
-      this._provider = new ethers.JsonRpcProvider(opts.provider)
-    } else if (opts.provider instanceof ethers.JsonRpcProvider) {
+    if (typeof opts.provider === 'string' && opts.provider.startsWith('http')) {
```

Comment:
This is fine, but just noting: we thus do not have websocket support. (Also does not seem that `fetchFromProvider` accepts this)

Reply:
Correct.  This only works over HTTP.  If anyone ever asks about websocket support, we can explore it.  I'm not sure if node's native `fetch` instance supports WS anyway.

Reply:
If ws are added in future you might need to use an isomorphic library or dependency inject it to get it to work in both client and node